Gems/PhysX/Code/Tests/EditorTestUtilities.cpp

```diff
@@ -70,6 +70,24 @@ namespace PhysXEditorTests
         }
     }
 
+    void PhysXEditorFixture::ConnectToPVD()
+    {
+        auto* debug = AZ::Interface<PhysX::Debug::PhysXDebugInterface>::Get();
+        if (debug)
+        {
+            debug->ConnectToPvd();
+        }
+    }
+
+    void PhysXEditorFixture::DisconnectFromPVD()
+    {
+        auto* debug = AZ::Interface<PhysX::Debug::PhysXDebugInterface>::Get();
+        if (debug)
+        {
+            debug->DisconnectFromPvd();
+        }
+    }
+
```

Comment:
These functions don't seem to be used anywhere in this PR. Why adding them?

Reply:
I used them for debugging and after removing the actual calls I thought these are useful to keep for any other PhysX.Editor test if we need to check what's going on via PVD.